recipes/acl.rb

```diff
@@ -6,23 +6,52 @@
 # Copyright 2014, Rackspace
 #
 
+firewall 'iptables' do
+  action :enable
+  provider Chef::Provider::FirewallIptables
+end
```

Comment:
Should we leave this to upstream? Or at least don't force a provider and/or action?